mmdet/visualization/local_visualizer.py

```diff
@@ -243,7 +281,7 @@ def _draw_panoptic_seg(self, image: np.ndarray,
                     'pad': 0.7,
                     'edgecolor': 'none'
                 }],
-                horizontal_alignment='center')
+                horizontal_alignments='center')
         return self.get_image()
 
     @master_only
```

Comment:
Is there a way to get the `drawn_img` in a direct way from `add_datasample` rather than by writing it to a driver then reading it? In video_demo.py, we need collect all `drawn_img` and save them to a mp4 file.

Reply:
supported